app.js

```diff
@@ -33,8 +35,10 @@ app.use((err, req, res, next) => {
     res.status(500).send({ msg: 'Internal server error' });
 });
 
+/*
 app.listen(8080, () => {
     console.log(`Listening on port 8080`);
 });
+*/
```

Comment:
So... yesterday I learned that if I don't comment this out when running jest/Supertest then it gradually breaks my internet connection. Who knew??